### DISCUSSION TITLE
Liked items

### DISCUSSION THREAD

**Discussion by jonathanantoine** [trakt/trakt-api#748] [Q&A]
Hello,

Is there a way to lower the volume of exchanged data and request only the trakt ids (and type) of liked items ?

Currently it returns whole objects.

Thanks !

**Comment by rectifyer** [comment]
In general, we don't have super minimal responses, but that could be a good idea to help reduce API response size for apps that standardize on using Trakt IDs internally.

Which specific API method are you referring to? What would an example response look like?

**Comment by jonathanantoine** [comment]
My app uses trakt as it's first data source and everything is bases on trakt's id. A lot of part of my "sync process" only requires trakts id. Here are the api enpoint I could think of :

- /collection/shows
- /users/hidden/
- /users/likes/comments
- /sync/watched/shows
- /users/{userName}/watchlist/shows

**Comment by nixgates** [comment]
I'd like to second this, internally my app uses Trakt ID's for organizing it's local database and I would certainly love to make use of super minimal calls where possible.

**Comment by kevincador** [comment]
I would use that kind of response too. Could be a _minimal_ Extended Info param.

Maybe it's a bit extreme but the responses could be just arrays of trakt ids (where possible) and something like (for `/sync/ratings`, where it's still usefull to have an associated value):
`{ "190":10, "54":5, "23":7 }`
or 
`[{ "trakt":190, "rating":10 }, { "trakt":54, "rating":5 }, { "trakt":23, "rating":7 }]`

Another solution would be to just send the same response for each endpoint that takes Extended Info but removing everything like (/sync/rating):
```
[
  {
    "rating": 10,
    "type": "movie",
    "movie": {
      "ids": {
        "trakt": 1
      }
    }
  }
]
```
